indy-vdr-proxy/Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde_json = "1.0"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "signal"] }
 tokio-rustls = { version = "0.24", optional = true }
 url = "2.2.2"
+sled = "0.34.7"
```

Comment:
It might be nice to expose a convenience wrapper in the indy_vdr crate so that the proxy doesn't also have to depend directly on sled.